lib/core/helpers/app_config_helper.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:convert';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:real_estate_app/core/models/app_config/AppConfig.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class AppConfigHelper {
+  Future<AppConfig> getAppInfo() async {
+    FirebaseFirestore fireStoreDB = FirebaseFirestore.instance;
```

Comment:
just  a suggestion, shoudn't the firebase call be in data layer?

Reply:
I didn't see the data layer